properties.go

```diff
@@ -562,6 +562,31 @@ func (p *Properties) String() string {
 	}
 	return s
 }
+func (p *Properties) FMap(f func(*Properties) interface{}) interface{} {
```

Comment:
i want a properties to return a datatype that only my users know I do not - thats what this may help for...